tests/testthat/test-install.R

```diff
@@ -26,6 +26,9 @@ test_that("install_cmdstan() successfully installs cmdstan", {
 
 test_that("install_cmdstan() errors if installation already exists", {
   skip_if_offline()
+  if (not_on_cran()) {
```

Comment:
this function doesn't exist anymore and so should cause an error!

Reply:
the only thing I can think of is maybe `skip_if_offline()` is being triggered? but that doesn't really make any sense if cmdstan was successfully installed from the internet in each of these CI runs. I will try removing `skip_if_offline()` just to see what happens.